CONTRIBUTING.md

```diff
@@ -65,6 +65,20 @@ the API interactions should be well understood and there should be no
 technical surprises; we expect the only discussion necessary on PRs to be
 for wordsmithing and formatting.
 
+## Github Labels
+
+There is a label for each phase of the SDLC:
+
+- `reviewing proposal`: The API working group is reviewing a proposed change
+to the API.
+- `proposal finalized`: (??)
+- `validating through implementation`: One or more platforms that integrate
+   with the API implement the proposed change to provide feedback on the design
+- `validated by implementation`: (??)
```

Comment:
These labels seem to be focused on the "issue" used to track the feature, right?  Should we close the issue once its been added to the "alpha features" doc? I guess it does seem redundant to keep the issue alive if we have it in the alpha-doc.  Although, `pr submitted` might be good to know that we have an open PR for the alpha-doc.